packages/common/src/ui/components/NavLink.tsx

```diff
@@ -55,24 +65,25 @@ export const AppNavLink: FC<ListItemLinkProps> = props => {
       )),
     [to]
   );
-  const className = clsx(
-    classes['drawerMenuItem'],
-    !!selected && classes['drawerMenuItemSelected']
-  );
 
   return (
-    <li>
-      <Tooltip disableHoverListener={drawer.isOpen} title={text || ''}>
-        <ListItem
-          selected={selected}
-          button
+    <Tooltip disableHoverListener={drawer.isOpen} title={text || ''}>
+      <StyledListItem isOpen={drawer.isOpen} isSelected={selected} to={to}>
+        <ListItemButton
+          sx={{
+            ...getListItemCommonStyles(drawer.isOpen),
+            '&.MuiListItemButton-root:hover': {
+              backgroundColor: 'transparent',
+            },
+          }}
+          disableGutters
```

Comment:
this was a bit of a nightmare.. using `styled` meant I was getting weird cryptic TS warnings and things. Decided to stop and just use the `sx` prop 🤷